src/cli/cli_app.js

```diff
@@ -441,13 +512,13 @@ class CLI {
       if (cmds[args[0]]) {
         cmds[args[0]](args);
       } else {
-        console.log('XXX no such command XXX ');
+        log.info('XXX no such command XXX ');
       }
     });
     return this;
   }
 }
 // TODO:: the CLI starts automatically for now
-const cli = new CLI().start();
+new CLI().start();
```

Comment:
This should be removed.. It happens twice now

Reply:
you right.

Reply:
made change in server.js